--- test/speedtest1.c.orig
+++ test/speedtest1.c
@@ -2996,7 +2996,10 @@
 
   /* "mix1" is a macro testset: */
   static char zMix1Tests[] =
-    "main,orm/25,cte/20,json,fp/3,parsenumber/25,rtree/10,star"
+    "main,orm/25,cte/20,json,fp/3,parsenumber/25,star"
+#ifdef SQLITE_ENABLE_RTREE
+    ",rtree/10"
+#endif
 #if !defined(SQLITE_SPEEDTEST1_WASM)
     ",app"
     /* This test misbehaves in WASM builds: sqlite3_open_v2() is
